src/EPR.Calculator.Frontend/Controllers/DashboardController.cs

```diff
@@ -80,7 +80,7 @@ private static List<DashboardViewModel> GetCalulationRunsData(List<CalculationRu
             {
                 foreach (var calculationRun in calculationRuns)
                 {
-                    var classification_val = runClassifications.FirstOrDefault(c => (int)c == calculationRun.CalculatorRunClassificationId);
+                    var classification_val = runClassifications.Find(c => (int)c == calculationRun.CalculatorRunClassificationId);
```

Comment:
I think this code with throw error if _classification_val_ is null. Please check.

Reply:
classification_val will be default value RunClassification.None